src/thirdparty/tests/opencascade_smoke.cpp

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "gtest/gtest.h"
+#include <iostream>
+
+#include <opencascade/Standard_Version.hxx>
```

Comment:
Nitpick: "<>" should only be used for system libraries.... I can be alone in my holy war...

Reply:
updated